### DISCUSSION TITLE
Is there a way to invert the y-axis?

### DISCUSSION THREAD

**Discussion by dkshin** [patrykandpatrick/vico#864] [Questions]
### Question

Is there a way to invert the y-axis? 
I need a bar chart with the y-axis inverted to show monthly ranking changes.
Is there a way to do this?

### Vico version(s)

2.0.0-alpha.28

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hello, @dkshin. Your question is vague. Can you elaborate on what you're looking to achieve?
